example/sankey-diagram.md

```diff
@@ -0,0 +1,33 @@
+# Simple Sankey Diagram
+Given an array of [nodes](http://d3plus.org/docs/#Sankey.nodes) and an array of [links](http://d3plus.org/docs/#Sankey.links), [d3plus-network](https://github.com/d3plus/d3plus-network) creates a simple sankey visualization.
```

Comment:
Add an empty line break between the title and the text. Here is some grammar edits:

```
Given an *Array* of [nodes](http://d3plus.org/docs/#Sankey.nodes) and an *Array* of [links](http://d3plus.org/docs/#Sankey.links), [d3plus-network](https://github.com/d3plus/d3plus-network) can create a Sankey visualization using the [Sankey](http://d3plus.org/docs/#Sankey) class.
```